.travis.yml

```diff
@@ -1,17 +1,28 @@
 language: generic
 sudo: required
+addons:
+  homebrew:
+    update: true
+    packages:
+      - autoconf
+      - gnu-getopt
+      - gnu-sed
+      - gnu-tar
+      - libtool
+      - ninja
+      - pkg-config
+      - python3
+      - swig
+      - wget
 matrix:
   include:
     - os: osx
       osx_image: xcode11.3
       install:
-        - brew update
-        - brew upgrade
-        - brew install ninja autoconf libtool gnu-sed python3 wget pkg-config swig gnu-getopt gnu-tar
         - python3 -m pip install virtualenv meson===0.51.2
         - curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain 1.42.0
       script:
-        - source $HOME/.cargo/env && ./tools/build.sh --clang --python-version 3.7 --enable-rust
+        - source $HOME/.cargo/env && ./tools/build.sh --clang --python-version $(python3 -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')") --enable-rust
```

Comment:
If we passed `--python-version 3` here, and thus `python3` to pymodule.find_installation in meson.build, it looks like meson should automatically discover the right python?

Reply:
meson couldn't discover its own shoes, sadly.

There can be multiple python3 versions installed, and versions 3.8+ require a hack that previous versions don't. So I think we have to keep this for now (or at least until meson is updated/we update, which no doubt will come with its own new problems).